### ISSUE TITLE
Markdown links in web app:  same tab?

### ISSUE THREAD

**Issue by lephtHanded** [binwiederhier/ntfy#866]
:question: **Question**
In my experience, links created by markdown aka [this type of link](https://github.com/binwiederhier/ntfy) usually open in new tabs within my browser.  
When I visit the NTFY web app (self hosted) the links open within the same tab.  Are the other sites that implement the MD links putting in additional behind the scenes HTML for that?

(I know I'm a lazy bum that doesn't want to shift+click or middle click :) )

**Comment by masterujjval** [comment]
Yeah because Markdown currently does not support hyperlinks being opened on a new tab.
Consider reading [this](https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown/8000171#8000171).

**Comment by lephtHanded** [comment]
thank you, great info.
